geomesa-ui/src/main/ui/src/app/map/map.js

```diff
@@ -1,39 +1,47 @@
 angular.module('geomesa.map', [])
 
-    .directive('geomesaMap', [function () {
+    .directive('geomesaMap', ['$http', function ($http) {
+
         return {
             restrict: 'E',
             scope: {
                 map: '=?',
                 api: '=',
-                selectedPoint: '='
+                selectedFeatures: '=',
             },
-            link: function (scope, element, attrs) {
-                var baseLayer = L.tileLayer.provider('Stamen.TonerLite'),
-                    //wmsLayer = L.tileLayer.wms("http://dgeo:8080/geoserver/sf/wms", {
-                    wmsLayer = L.tileLayer.wms("localhost:8080/wcs/cite/wms", {
-                        layers: 'cite::QuickStart',
-                        format: 'image/png',
-                        transparent: true
-                    });
 
-                scope.map = L.map(element[0], {
-                    //center: L.latLng(40.279957,-74.73862),
-                    center: L.latLng(-38.09, -76.85),
-                    zoom: 8,
-                    maxZoom: 18,
-                    minZoom: 3,
-                    attributionControl: false,
-                    layers: [baseLayer, wmsLayer]
-                });
+            link: function (scope, element, attrs) {
+                scope.selectedFeatures = [
+                    {
+                        Start: ' Click a data point to view its attributes.'
+                    }
+                ];
 
-                scope.map.on('click', function (evt) {
-                    scope.$apply(function () {
-                        scope.selectedPoint = {
-                            lat: evt.latlng.lat,
-                            lng: evt.latlng.lng
-                        };
+                var projection = ol.proj.get('EPSG:3857'),
+                    extent = projection.getExtent(),
```

Comment:
I definitely agree that these things should be configurable. I am going to work on the temporal density iterator for a bit and will get back to you on this. Some initial thoughts would be, the only two sources of configuration I can think of would be to either get configuration info passed to us in the scope, or to get it via a service. Services are singletons and this would seem to restrict us to only one map instance. Who knows, maybe some day we will want to have two maps side by side so we can compare the two. So maybe we should get configuration info via the scope. If we get configuration info via the scope, I guess we'll need to decide what goes in the link function vs what we get passed to us via the scope. Should we just get everything from the scope and have an empty link function? I need to take some time to think about all of this, but I thought I'd let you know where my thoughts are starting.